packages/csv-parser-discount-code/src/main.js

```diff
@@ -0,0 +1,101 @@
+/* @flow */
+import type {
+  LoggerOptions,
+  ParseOptions,
+  ParserSummary,
+ } from 'types/discountCodes'
+import _ from 'lodash'
+import csv from 'csv-parser'
+import JSONStream from 'JSONStream'
+import highland from 'highland'
+import npmlog from 'npmlog'
+import { unflatten } from 'flat'
+
+export default class CsvParser {
```

Comment:
let's make this more descriptive. Can be confusing with the `csv-parser` module above.